conftest.py

```diff
@@ -7,6 +8,50 @@
 
 log = logging.getLogger(__name__)
 
+@dataclass
+class SensorInfo:
+    name: str
+    hid: str
+    model: str
+    firmware_version: int
+    reading_interval: float
```

Comment:
This type hint should read `int` instead of `float`. Just noticed it, sorry I didn't catch it yesterday.

Reply:
Thank You!
I should have also chacked it((